### DISCUSSION TITLE
using own data instead of sample app

### DISCUSSION THREAD

**Discussion by dib789** [Project-MONAI/MONAILabel#1082] [Q&A]
Hello, 
We are Master students working on Monai label. We followed the tutorials (Radiology app/Spleen example) and want to try it out with our own dataset. How can we load our data (e.g. CT images) into the Monai label server using docker in order to perform segmentation and model training?
Thank you

**Comment by diazandr3s** [comment]
Hi @dib789,

Thanks for your interest in MONAI Label.

You may find this discussion useful: https://github.com/Project-MONAI/MONAILabel/discussions/1055#discussioncomment-3830237

When using Docker, you need to mount the local folder that contained the images into the Docker container: https://docs.docker.com/storage/bind-mounts/

Also, I wanted to make you aware of this video tutorial: https://www.youtube.com/watch?v=8y1OBQs2wis&list=PLtoSVSQ2XzyD4lc-lAacFBzOdv5Ou-9IA

Hope this helps,
